### PR TITLE
Brand Management Layout

### PR DESCRIPTION
**Pull request by rafimarc101**

### CODE REVIEW

File: src/components/BrandManagementComponents/MainBrandManagement.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    <b-container fluid class="whiteWrapper">
+        <b-row>
+            <b-col sm="12">
+                <b-container fluid class="appBackgroundShadeColor mainItemContainer">
+                    <b-row class="containerWrapper">
+                        <b-col class="whiteWrapper" sm="5">
+                            <b-container fluid class="searchColumn">
+                                <b-row>
+                                    <b-col sm="12">
+                                        <b-row>
```
Comment (Cell5) [inline src/components/BrandManagementComponents/MainBrandManagement.vue]: please consider the Bootstrap Grid rule https://bootstrap-vue.js.org/docs/components/layout#rows-b-row-and-b-form-row
the b-row must be inside b-container

File: src/components/BrandManagementComponents/MainBrandManagement.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                                            <b-col sm="12">
+                                                <SearchField/>
+                                            </b-col>
+                                        </b-row>
+                                    </b-col>
+                                </b-row>
+
+                                <b-row>
+                                    <b-col sm="12">
+                                        <b-row>
```
Comment (Cell5) [inline src/components/BrandManagementComponents/MainBrandManagement.vue]: the same issue as above

File: src/components/BrandManagementComponents/MainBrandManagement.vue (hunk 1 above)
Comment (epsotto) [reply src/components/BrandManagementComponents/MainBrandManagement.vue]: Rainier removed the redundant declarations.

File: src/components/BrandManagementComponents/MainBrandManagement.vue (hunk 2 above)
Comment (epsotto) [reply src/components/BrandManagementComponents/MainBrandManagement.vue]: Rainier removed the redundant declarations.

**Pull request merged**